WooCommerce/Classes/ViewRelated/Themes/ThemesCarouselView.swift

```diff
@@ -0,0 +1,130 @@
+import SwiftUI
+import Kingfisher
+
+
+struct ThemesCarouselView: View {
+    /// Tuple array containing theme name and theme Url string pair.
+    let themesInfo: [(name: String, url: String)] = [
+        (name: "Google", url: "google.com"),
+        (name: "Facebook", url: "facebook.com"),
+        (name: "WordPress", url: "wordpress.com"),
+        (name: "Microsoft", url: "microsoft.com"),
+        (name: "Amazon", url: "amazon.com"),
+        (name: "Apple", url: "apple.com")
+    ]
+
+    /// The title of the message at the end of the carousel.
+    let lastMessageHeading: String
+
+    /// The content of the message at the end of the carousel.
+    let lastMessageContent: String
+
+
+    /// Scale of the view based on accessibility changes
+    @ScaledMetric private var scale: CGFloat = 1.0
+
+    var body: some View {
+        ScrollView(.horizontal, showsIndicators: false) {
+            HStack() {
+
+                // Demo for correct state
+                ForEach(themesInfo, id: \.name) { theme in
+                    if let themeUrl = getThemeUrl(themeUrl: theme.url) {
+                        ThemeImage(url: themeUrl)
+                    }
+                }
+
+                // Demo for error state
+                ThemeName(name: themesInfo[0].name)
+
+                // Demo for last message content
+                VStack {
+                    Text(lastMessageHeading)
+                        .bold()
+                        .bodyStyle()
+                        .padding(.horizontal, Layout.contentPadding)
+
+                    Spacer()
+                        .frame(height: Layout.contentPadding)
+
+                    Text(lastMessageContent)
+                        .subheadlineStyle()
+                            .multilineTextAlignment(.center)
+                        .padding(.horizontal, Layout.contentPadding)
+                }.frame(
```

Comment:
Instead of adding spacer with fixed height, we should add spacing to the outer `VStack` to simplify the layout:

```suggestion
                VStack(spacing: Layout.contentPadding) {
                    Text(lastMessageHeading)
                        .bold()
                        .bodyStyle()
                        .padding(.horizontal, Layout.contentPadding)

                    Text(lastMessageContent)
                        .subheadlineStyle()
                            .multilineTextAlignment(.center)
                        .padding(.horizontal, Layout.contentPadding)
                }.frame(
```